migrations/versions/e47fde01cc62_add_csm_sequence.py

```diff
@@ -0,0 +1,23 @@
+"""add csm sequence
+
+Revision ID: e47fde01cc62
+Revises: e505f2e15499
```

Comment:
I don't think you need an extra migration to merge the heads. I think the issue is that the head this 'Revises' is the same as the head 'Revises' [here](https://github.com/sanger/baracoda/pull/300/files#diff-3a4c17914d8b2a7467169528dbf78cc7d529b34da408a3181f784357d84763d9R4). You should change them to be sequential (change one to revise from the others revision ID). Think you will also need to change the down_revision to be the same.